data/json/recipes/recipe_armor.json

```diff
@@ -8339,17 +8339,63 @@
       [ "hat_hunting", 1 ]
     ],
     [
+      [ "mask_dust", 1 ],
+      [ "bandana", 2 ]
+    ],
+    [
       [ "glasses_safety", 1 ],
       [ "glasses_bal", 1 ]
     ],
     [
-      [ "leather", 12 ],
+      [ "leather", 8 ],
       [ "tanned_hide", 2 ]
     ]
   ]
 },
 {
   "type" : "recipe",
+  "result": "helmet_scavenger",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_HEAD",
+  "skill_used": "tailor",
+  "skills_required" : ["fabrication", 3],
+  "difficulty": 8,
+  "time": 40000,
+  "reversible": false,
+  "autolearn": true,
+  "qualities" : [
+    {"id":"CUT","level":1,"amount":1}
+  ],
+  "tools": [
+    [
+      [ "soldering_iron", 8 ],
+      [ "toolset", 8 ]
+    ],
+    [
+      [ "needle_bone", 65 ],
+      [ "needle_wood", 65 ],
+      [ "sewing_kit", 65 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "hat_ball", 1 ],
+      [ "hat_hunting", 1 ]
+    ],
+    [
+      [ "mask_filter", 1 ]
+    ],
+    [
+      [ "glasses_bal", 1 ],
+      [ "goggles_ski", 1 ]
```

Comment:
Weak spot in current crafting: filter mask + ski goggles (effectively) = gas mask, but we can't support letting the gas mask stand in w/o a separate recipe.  Not a problem, just a teachable moment.